elated-egrets/bangazon-api
# J ruser

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git pull JRuser
git checkout JRuser
python manage.py runserver
```

## Steps to test
pull down branch
open in vscode
check and make sure file structure is right
make sure the server runs correctly and opens up the API
## What you changed
added user model, serialization, and view
also had to add the virtual env to the gitignore
## What was added to read me
going to add the user model and view and what it entails
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
1. Has the readme been updated to reflect changes made?
1. Has a teammate approved the PR?

Then Ship It!
